### DISCUSSION TITLE
error in npm run build

### DISCUSSION THREAD

**Discussion by abdulnasser-hamayseh** [vercel/next.js#41177] [Help]
i want to uplode my next app to cpanel
when using 
npm run build 
after added server.js page 
and update package.json 
as in this page https://nextjs.org/docs/advanced-features/custom-server
return warning  "next" should not be imported directly,
and error 
TypeError: next is not a function
not complete npm run build

**Comment by icyJoseph** [comment]
Are you running `npm install` or `npm ci`? Also which Node.js version is your CPanel running, you need at least 14 IIRC.
